promtail/tasks/main.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: Install Deps
+  become: true
+  ansible.builtin.apt:
+    name:
+      - apt-transport-https
+      - curl
+      - gnupg
+      - python3-apt
+    state: present
+    update_cache: true
+    cache_valid_time: 300
+  register: apt_action
+  retries: 100
+  until: apt_action is success or ('Failed to lock apt for exclusive operation' not in apt_action.msg and '/var/lib/dpkg/lock' not in apt_action.msg)
+
+- name: Create the apt keyrings directory
```

Comment:
Dont know if it matters too much, but we use a different path for the chia keychain - might be nice to be consistent https://github.com/Chia-Network/ansible-roles/blob/main/chia-apt-key/tasks/main.yml#L21

Reply:
changed the path